tests/ctests/test_utils.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <stdio.h>
+
+#include "types.h"
+#include "netplan.h"
+#include "parse.h"
+
+NetplanState *
+load_fixture_to_netplan_state(const char* filename)
+{
+
+    GError *error = NULL;
```

Comment:
```suggestion
    g_autoptr(GError) error = NULL;
```

We can make use of some glib goodies here, as it is included via "types.h". Could also use `g_autofree gchar* filepath` and/or `filepath = g_build_path(G_DIR_SEPARATOR_S, FIXTURESDIR, filename, NULL);` below. Or at least clear the GError at the end.